app/uk/gov/hmrc/tai/service/journeyCache/JourneyCacheService.scala

```diff
@@ -133,4 +134,7 @@ class JourneyCacheService @Inject()(val journeyName: String, journeyCacheConnect
 
   def flushWithEmpId(empId: Int)(implicit hc: HeaderCarrier): Future[TaiResponse] =
```

Comment:
can we not just use this?

Reply:
This is just to keep the test ones separate